lib/howl/buffer.moon

```diff
@@ -357,6 +371,12 @@ with config
     default: true
     type_of: 'boolean'
 
+  .define
+    name: 'backup_files'
+    description: 'Whether or not to make temporary backups of files while saving'
+    default: true
```

Comment:
I think the default should be `false`, mainly due to privacy concerns. Copying every file saved into another directory shouldn't happen automatically. Consider I may be viewing/editing a sensitive file in a network folder or usb drive - that shouldn't be copied into backupdir.